src/js/brushing/mg_line_brushing.js

```diff
@@ -0,0 +1,345 @@
+(function(root, factory) {
```

Comment:
Please ignore this file and have a look at `line.js` below. I guess there have been a few indentation errors, I'll correct them first thing tomorrow morning :+1: